app/src/components/features/rules/DuplicateRuleModal/index.tsx

```diff
@@ -27,10 +40,32 @@ const DuplicateRuleModal: React.FC<Props> = ({
   rule,
   onDuplicate,
 }) => {
+  const navigate = useNavigate();
+  const availableWorkspaces: TeamWorkspace[] = useSelector(getAvailableTeams);
+  const currentlyActiveWorkspace = useSelector(getCurrentlyActiveWorkspace);
   const appMode = useSelector(getAppMode);
   const [newRuleName, setNewRuleName] = useState<string>();
+  const [selectedWorkspaceId, setSelectedWorkspaceId] = useState<string>(null);
   const ruleNameInputRef = useRef<InputRef>(null);
 
+  const isUsingWorkspaces = useMemo(() => {
+    return currentlyActiveWorkspace && availableWorkspaces?.length > 0;
```

Comment:
`currentlyActiveWorkspace` initial state (initially it's private workspace) is equal to

```  
currentlyActiveWorkspace: {
    id: null,
    name: null,
    membersCount: null,
  }
```
and `id`, `name` & `membersCount` are set upon switching to a workspace. So currentlyActiveWorkspace is always going to be defined